catalog/spatial/wfs/1.0.0/spatial-wfs-v1_0_0-source/src/test/java/org/codice/ddf/spatial/ogc/wfs/v1_0_0/catalog/source/TestWfsFilterDelegate.java

```diff
@@ -1,2116 +0,0 @@
-/**
- * Copyright (c) Codice Foundation
- * <p>
- * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
- * General Public License as published by the Free Software Foundation, either version 3 of the
- * License, or any later version.
- * <p>
- * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
- * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
- * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
- * is distributed along with this program and can be found at
- * <http://www.gnu.org/licenses/lgpl.html>.
- **/
-package org.codice.ddf.spatial.ogc.wfs.v1_0_0.catalog.source;
-
-import static org.mockito.Mockito.mock;
-
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-
-import org.codice.ddf.spatial.ogc.wfs.catalog.common.FeatureMetacardType;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-public class TestWfsFilterDelegate {
```

Comment:
@lessarderic I would like to point out though that there are two files with this name, one is ddf/spatial/ogc/wfs/v2_0_0/catalog/source/TestWfsFilterDelegate.java
and the other is
ddf/spatial/ogc/wfs/v1_0_0/catalog/source/TestWfsFilterDelegate.java

The deleted file we are looking at is in v1_0_0. The file in v2_0_0, however, is being used and has no commented out code. It might be that v1_0_0 is replaced with v2_0_0. If that is the case, maybe it should stay deleted?